app/src/shared/components/Layout/Nav.jsx

```diff
@@ -1,7 +1,7 @@
 import React, { Fragment, useContext } from 'react'
 import styled, { css, ThemeProvider, ThemeContext } from 'styled-components'
 import { useSelector } from 'react-redux'
-import { Translate, I18n } from 'react-redux-i18n'
+import { Translate } from 'react-redux-i18n'
```

Comment:
`Translate` outside the scope of this PR?

Reply:
Yep, I'll do the `shared` folder separarately 👍